src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -99,54 +101,83 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
 
     const hotInstanceColHeaders: string[] = (newStudentsHOTInstance.getColHeader() as string[]);
 
-    const studentsEnrollRequest: StudentsEnrollRequest = {
-      studentEnrollRequests: [],
-    };
+    const allStudentChunks: StudentEnrollRequest[][] = [];
+    let currentStudentChunk: StudentEnrollRequest[] = [];
```

Comment:
either put together consts or add line break between logical blocks of init code? purely aesthetics.